src/components/Layout.tsx

```diff
@@ -3,9 +3,14 @@ import { useRouter } from "next/router";
 import React from "react";
 import styled from "styled-components";
 
+import { ClickAwayListener } from "./display/fewlines/ClickAwayListener";
 import { Header } from "./display/fewlines/Header/Header";
+import { MobileNavigationBar } from "./display/fewlines/MobileNavigationBar/MobileNavigationBar";
 
 export const Layout: React.FC = ({ children }) => {
+  const [mobileNavBarOpen, setMobileNavbarOpen] = React.useState<boolean>(
```

Comment:
Can you replace the naming with `isMobileNavBarOpen`?

Reply:
OK fixed f7805ce